Application/Setting/User/Account/Service/Entity/TblUserAccount.php

```diff
@@ -240,4 +262,75 @@ public function setGroupByCount($GroupByCount)
     {
         $this->GroupByCount = $GroupByCount;
     }
+
+    /**
+     * @return string
+     */
+    public function getAccountCreator()
+    {
+        return $this->AccountCreator;
+    }
+
+    /**
+     * @param string $AccountCreator
+     */
+    public function setAccountCreator($AccountCreator = '')
+    {
+        $this->AccountCreator = $AccountCreator;
+    }
+
+    /**
+     * @return string
+     */
+    public function getAccountUpdater()
+    {
+        return $this->AccountUpdater;
+    }
+
+    /**
+     * @param string $AccountUpdater
+     */
+    public function setAccountUpdater($AccountUpdater = '')
+    {
+        $this->AccountUpdater = $AccountUpdater;
+    }
+
+    /**
+     * @return bool|string
+     */
+    public function getUpdateDate($format = 'd.m.Y H:i:s')
+    {
+
+        /** @var DateTime $UpdateDate */
+        $UpdateDate = $this->UpdateDate;
+        if ($UpdateDate instanceof DateTime) {
+            return $UpdateDate->format($format);
+        }
+        return false;
+    }
+
+    /**
+     * @param null|DateTime
+     */
+    public function setUpdateDate($UpdateDate = null)
+    {
+
+        $this->UpdateDate = $UpdateDate;
+    }
+
+    /**
+     * @return null|int
+     */
+    public function getUpdateType()
+    {
+        return $this->UpdateType;
+    }
+
+    /**
+     * @param null|int $UpdateType
+     */
+    public function setUpdateType($UpdateType = null)
```

Comment:
Codacy found an issue: [Incorrect spacing between argument "$UpdateType" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)